common/beerocks/bwl/CMakeLists.txt

```diff
@@ -9,8 +9,10 @@ file(GLOB_RECURSE bwl_common_sources ${MODULE_PATH}/common/*.c*)
 # Support more permissive c++ language
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11")
 
-# Default defaults
-set(BWL_TYPE "DUMMY")
+# Defaults
+if (NOT DEFINED BWL_TYPE)
+    set(BWL_TYPE "DUMMY")
+endif()
```

Comment:
This should be approached differently, like this (skeleton code):
```
set(DEFAULT_BWL_TYPE "DUMMY")
if (TARGET_PLATFORM STREQUAL "rdkb")
    set(DEFAULT_BWL_TYPE "DWPAL")
...
endif()
set(BWL_TYPE ${DEFAULT_BWL_TYPE} CACHE STRING "Which BWL backend to use")
```

Also, it should be done in a separate commit from the rest of the changes.